radicle-cli/src/commands/patch/update.rs

```diff
@@ -5,9 +5,10 @@ use radicle::git;
 use radicle::prelude::*;
 use radicle::storage::git::Repository;
 
+use crate::terminal as term;
+
 use super::common::*;
 use super::Options;
-use crate::terminal as term;
```

Comment:
`crate` and `super` are both local crate imports, so can stay in the same grouping